content/docs/reference/design-docs/conflict-detection/index.en.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Conflict detection"
+linkTitle: "Conflict detection"
+description: "Detect unrealistic timetables"
+weight: 45
+---
+
+{{% pageinfo color="warning" %}}
+This document is a work in progress
+{{% /pageinfo %}}
+
+Conflict detection is a process which enables figuring out if a timetable,
+comprised of many train paths, can be performed as planned.
+
+If a train has to slow down because of another train, or undergoing work on
+rail infrastructure, these actors are in conflict, which causes the timetable
+to become impossible to perform.
+
+Conflict detection relies on interlocking and signaling simulation to:
+1) figure out what each actor requires to perform its duty undisturbed
+2) detect conflicting requirements
+
+This system has to:
+ - produce conflicts which can be linked back to a root cause
+ - operate in way that can be visualized and interpreted
+ - be scalable: it should work just as fast given a massive number of train paths to intersect
+ - enable threading new train paths into an existing timetable
+
+The design of this system is guided by a number of constraints:
+ - **it must not cause false-negatives**: if no conflicts are detected, a multi-train simulation of the same timetable must no yield any slowdowns
+ - it cannot rely on data we do not have
+ - it has to enable later support of mobile block systems
+ - it has to rely on existing signaling and interlocking simulation
+ - it has to enable detecting conflicts regardless of the signaling system in use
+ - support transitions between signaling systems
+ - support conflicts between different signaling systems
```

Comment:
```suggestion
 - it has to support transitions between signaling systems
 - it has to support conflicts between different signaling systems
```
Otherwise those two bullet points are a bit weird syntaxically